auslib/blobs/base.py

```diff
@@ -123,3 +123,6 @@ def getFooterXML(self, updateQuery, update_type, whitelistedDomains, specialForc
 
     def getInnerXML(self, updateQuery, update_type, whitelistedDomains, specialForceHosts):
         raise NotImplementedError()
+
+    def containsForbiddenDomain(self, product, whitelistedDomains):
+        return False
```

Comment:
I think it would be best to raise NotImplementedError here. Defaulting to False makes it easier for us to introduce bugs that would allow bad domains through in the future.